actions/gha-workflow-validator/src/strings.ts

```diff
@@ -0,0 +1,57 @@
+export const COMMENT_HEADER = `### GHA Workflow Validator Results`
+
+const FIXING_ERRORS = `
```

Comment:
I would try to use github's job summaries: https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/
or the notice annotation here instead

Reply:
Added summaries :+1: